qa/smoke-test-plugins/build.gradle

```diff
@@ -9,26 +9,32 @@
 
 import org.apache.tools.ant.filters.ReplaceTokens
 
-apply plugin: 'elasticsearch.legacy-yaml-rest-test'
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
 
 ext.pluginPaths = []
+ext.pluginNames = []
 project(':plugins').getChildProjects().each { pluginName, pluginProject ->
   pluginPaths << pluginProject.path
+  pluginNames << pluginName
 }
 
-testClusters.matching { it.name == "yamlRestTest" }.configureEach {
+dependencies {
+  clusterModules project(":modules:lang-painless")
   pluginPaths.each { pluginPath ->
-    plugin pluginPath
+    clusterPlugins(project(pluginPath))
   }
-  setting 'xpack.security.enabled', 'false'
 }
 
 ext.expansions = [
   'expected.plugins.count': pluginPaths.size()
 ]
 
-tasks.named("processYamlRestTestResources").configure {
+tasks.named("processYamlRestTestResources") {
```

Comment:
I think you trigger early creation of the task here

Reply:
Yeah. The `configure()` call isn't necessary when calling `named` or `register` in most places. I've been slowly removing them to clean things up a bit as I encounter them.

Reply:
👍